docs/sections/user_guide/cli/drivers/mpas_init.rst

```diff
@@ -0,0 +1,107 @@
+``mpas_init``
+=============
+
+The ``uw`` mode for configuring and running the MPAS init_atmosphere component. Any of the ``TASK`` arguments listed below may be called to generate that component required for the ultimate ``run`` task. A ``provisioned_run_directory`` will perform all of the tasks except for running the executable.
```

Comment:
Perhaps consider whether any of the suggestions made for the `mpas` driver also apply to this one.